api/proto/teleport/autoupdate/v1/autoupdate.proto

```diff
@@ -71,14 +71,17 @@ message AgentAutoUpdateSchedules {
 
 // AgentAutoUpdateGroup specifies the update schedule for a group of agents.
 message AgentAutoUpdateGroup {
+  reserved 4;
+  reserved "wait_days";
```

Comment:
+1 for the clarity of changing from days to an unambiguous time unit. Nicely done.